apps/preview/next/pages/showcases/Banners/Hero.tsx

```diff
@@ -0,0 +1,41 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfButton } from '@storefront-ui/react';
+import headphones from '@assets/hero-headphones.png';
+import mobileBackground from '@assets/hero-bg-mobile.png';
+import desktopBackground from '@assets/hero-bg.png';
+
+const variables = {
+  '--sf-img-mobile': `url(${mobileBackground.src})`,
+  '--sf-img-desktop': `url(${desktopBackground.src})`,
+} as React.CSSProperties;
+
+export default function Hero() {
+  return (
+    <div
+      style={variables}
+      className="md:flex md:flex-row-reverse min-h-[600px] max-w-[1536px] mx-auto bg-[url:var(--sf-img-mobile)] md:bg-[url:var(--sf-img-desktop)] bg-center bg-cover bg-no-repeat"
```

Comment:
Is this `flex-row-reverse` solution made intentionally? If not maybe it could be easier just to switch the elements inside, wdyt?

Reply:
yes, it's intentionally :)

Reply:
Yes, you are right. That was a silly question, but the designs are somewhat misleading for me, there are two versions on desktop and one for mobile. So now I knwo, that no matter if the image is on left or right it should be on top when in mobile mode :)